internal/scaffold/olm-catalog/csv.go

```diff
@@ -237,6 +237,24 @@ func (s *CSV) setCSVDefaultFields(csv *olmapiv1alpha1.ClusterServiceVersion) {
 			{Type: olmapiv1alpha1.InstallModeTypeAllNamespaces, Supported: true},
 		}
 	}
+	if len(csv.Spec.Icon) == 0 {
+		emptyIcon := olmapiv1alpha1.Icon{
+			Data:      "",
+			MediaType: "",
+		}
+		csv.Spec.Icon = []olmapiv1alpha1.Icon{emptyIcon}
+	}
+	if len(csv.Spec.Keywords) == 0 {
+		csv.Spec.Keywords = []string{""}
+	}
+	if len(csv.Spec.Maintainers) == 0 {
+		emptyMaintainers := olmapiv1alpha1.Maintainer{
+			Name:  "",
+			Email: "",
+		}
+		csv.Spec.Maintainers = []olmapiv1alpha1.Maintainer{emptyMaintainers}
```

Comment:
This will automatically construct an empty Icon:
```suggestion
		csv.Spec.Maintainers = make([]olmapiv1alpha1.Maintainer, 1)
```
And remove the above 4 lines.